pages/windows/vswhere.md

```diff
@@ -0,0 +1,20 @@
+# vswhere
+
+> Locate Visual Studio 2017 and newer installations.
+> More information: <https://github.com/microsoft/vswhere>.
+
+- Find the path of vcvarsall.bat to set environment variables:
+
+`vswhere -products * -latest -prerelease -find "**/VC/Auxiliary/Build/vcvarsall.bat"`
+
+- Find the directory of the x64 MSVC compiler (cl.exe, etc):
+
+`vswhere -products * -latest -prerelease -find **/Hostx64/x64/*`
```

Comment:
```suggestion
`vswhere -products {{*}} -latest -prerelease -find {{**/Hostx64/x64/*}}`
```
Also, why did you use `" "` in the first example, but not here?

Reply:
These are not variables that you want to add `{{}}` around them.